test/controllers/medias_controller_test.rb

```diff
@@ -255,40 +238,13 @@ def setup
     assert_equal 'Timeout', JSON.parse(@response.body)['data']['error']['message']
   end
 
-  test "should return API limit reached error" do
-    Twitter::REST::Client.any_instance.stubs(:user).raises(Twitter::Error::TooManyRequests)
-    Twitter::Error::TooManyRequests.any_instance.stubs(:rate_limit).returns(OpenStruct.new(reset_in: 123))
-
-    authenticate_with_token
-    get :index, url: 'https://twitter.com/anxiaostudio', format: :json
-    assert_response 429
-    assert_equal 123, JSON.parse(@response.body)['data']['message']
-
-    Twitter::REST::Client.any_instance.unstub(:user)
-    Twitter::Error::TooManyRequests.any_instance.unstub(:rate_limit)
-  end
-
   test "should render custom HTML if provided by parser" do
```

Comment:
@daniel-shuy why did you remove these tests? They were not affected by your changes, they are also passing.
Could you add them back, please?